Attendee.js

```diff
@@ -0,0 +1,43 @@
+// The Attendee class can be utililized to keep certain information of the attendee, 
+// a class for the user needed to be created in addition to the session class. 
+// The test for this JS class file utilizes Jest.IO
+class Attendee {
+    constructor() {
+        this.sessionId;
+        this.screenName;
+        this.TimeLastPolled;
```

Comment:
Style: Properties must be documented and private properties should have an underscore after.

https://google.github.io/styleguide/jsguide.html#jsdoc-property-comments